SCons/Node/FSTests.py

```diff
@@ -1583,44 +1578,40 @@ def test_remove_fail(self) -> None:
 
     def test_drive_letters(self) -> None:
         """Test drive-letter look-ups"""
-
         test = self.test
-
         test.subdir('sub', ['sub', 'dir'])
 
+        seps = [os.sep]
+        if os.sep != '/':
+            seps.append('/')
+
         def drive_workpath(dirs, test=test):
             x = test.workpath(*dirs)
             drive, path = os.path.splitdrive(x)
             return 'X:' + path
 
         wp = drive_workpath([''])
-
-        if wp[-1] in (os.sep, '/'):
+        if wp[-1] in seps:
             tmp = os.path.split(wp[:-1])[0]
         else:
             tmp = os.path.split(wp)[0]
-
         parent_tmp = os.path.split(tmp)[0]
         if parent_tmp == 'X:':
             parent_tmp = 'X:' + os.sep
-
         tmp_foo = os.path.join(tmp, 'foo')
-
         foo = drive_workpath(['foo'])
         foo_bar = drive_workpath(['foo', 'bar'])
-        sub = drive_workpath(['sub', ''])
-        sub_dir = drive_workpath(['sub', 'dir', ''])
-        sub_dir_foo = drive_workpath(['sub', 'dir', 'foo', ''])
-        sub_dir_foo_bar = drive_workpath(['sub', 'dir', 'foo', 'bar', ''])
-        sub_foo = drive_workpath(['sub', 'foo', ''])
+        # The following are not used, but kept for reference.
```

Comment:
Why remove these?

Reply:
I'm not sure I understand what you mean by "not in use at all"?
Is it possible they could be?
Does it exercise the logic in a way not otherwise exercised?

Reply:
Simple case of variables set but never used (in the existing code). I commented them out rather than removing, but I'm unlikely to go back and write any code for them